conf/layer.conf

```diff
@@ -9,5 +9,5 @@ BBFILE_PRIORITY_snapd = "7"
 # This should only be incremented on significant changes that will
 # cause compatibility issues with other layers
 LAYERVERSION_snapd = "1"
-LAYERSERIES_COMPAT_snapd = "kirkstone langdale mickledore"
+LAYERSERIES_COMPAT_snapd = "nanbield"
```

Comment:
while technically we have a dedicated kirkstone branch, I don't think there are any changes what would make the current master incompatible with kirkstone and later releases

Reply:
Dropped